### DISCUSSION TITLE
Traslation to OWASP MASTG

### DISCUSSION THREAD

**Discussion by jsectools** [OWASP/mastg#2476] [Ideas]

**Comment by jsectools** [comment]
does not apply, since it is explained on the following website https://mas.owasp.org/contributing/1_How_Can_You_Contribute/#proof-reading

**Comment by cpholguera** [comment]
Hi @jsectools, I've just realized that that part of the website is outdated. 

We have had over a dozen translations of the MASVS and a few of the MASTG in the past, but we decided a few years ago to stop because the overhead is too much, as we have to contact dozens of people for each change, and the MASTG just changes too often and it will be impossible to keep it in sync through volunteer work. Also, the amount of content for the MASTG is too much to be translated by volunteers.


See the official statement here: https://mas.owasp.org/MASVS/

> Starting with MASVS v2.0.0, translations will no longer be included to focus on the development of MASTG v2.0.0. We encourage the community to create and maintain their own translations. Thank you to all the past translators who generously volunteered their time and expertise to make the MASVS accessible to non-English speaking communities. We truly appreciate your contributions and hope to continue working together in the future. The past MASVS v1.5.0 translations are still available in the MASVS repo.
